src/main/java/org/webshar/hrms/service/EmployeeService.java

```diff
@@ -33,9 +33,9 @@ public Employee getEmployeeById(Long id) throws EntityNotFoundException
   }
 
 
-  public Employee getEmployeeByEmployeeId(Long employeeId) throws EntityNotFoundException
+  public Employee getEmployeeByEmployeeId(String employeeId) throws EntityNotFoundException
```

Comment:
This method is never used. Kindly remove that. Also method logic is also not correct.